client/components/gsuite/gsuite-new-user-list/index.tsx

```diff
@@ -15,6 +15,7 @@ import './style.scss';
 interface Props {
 	autoFocus?: boolean;
 	children?: ReactNode;
+	dirty?: boolean;
```

Comment:
Unless this is driven by our validation framework, I would suggest naming this boolean variable `isDirty`.